wiki/Legal/Privacy/id.md

```diff
@@ -100,7 +98,7 @@ Sebagai seorang pengguna, Anda memiliki hak untuk memigrasi, memperbarui, atau m
 
 Dalam kebanyakan kasus, kiriman pengguna seperti postingan forum dan beatmap dapat dihapus secara tersendiri. Anda akan menemui tombol hapus yang terasosiasi langsung dengan persoalan yang bisa dihapus.
 
-Jika Anda tinggal di daerah dengan cakupan Uni Eropa, Anda memiliki opsi untuk menghapus akun Anda dari layanan kami. Harap diperhatikan bahwa proses ini dilakukan secara manual dan membutuhkan waktu beberapa hari untuk menyelesaikannya ([hubungi kami](mailto:privacy@ppy.sh) untuk mengajukan permintaan). Dalam kasus penghapusan akun, sebagian kontribusi Anda yang dilakukan secara publik mungkin tidak terhapus, seperti yang telah dijelaskan dalam "Informasi yang kami kumpulkan."
+Anda memiliki opsi untuk menghapus akun Anda dari layanan kami. Harap diperhatikan bahwa proses ini dilakukan secara manual dan membutuhkan waktu beberapa hari untuk menyelesaikannya ([hubungi kami](mailto:privacy@ppy.sh) untuk mengajukan permintaan). Dalam kasus penghapusan akun, sebagian kontribusi Anda yang dilakukan secara publik mungkin tidak terhapus, seperti yang telah dijelaskan dalam "Informasi yang kami kumpulkan."
```

Comment:
```suggestion
Anda memiliki opsi untuk menghapus akun milik sendiri dari layanan kami. Harap diperhatikan bahwa proses ini dilakukan secara manual dan membutuhkan waktu beberapa hari untuk menyelesaikannya. Anda dapat [menghubungi kami](mailto:privacy@ppy.sh) untuk mengajukan permintaan penghapusan. Dalam kasus penghapusan akun, sebagian kontribusi Anda yang telah dilakukan secara publik mungkin tidak terhapus, seperti yang telah dijelaskan dalam "Informasi yang kami kumpulkan."
```

Reply:
this may be a good proportion to Indonesia language

Reply:
Separating it into new sentence is better than the current one. 😄